src/debugger-configuration.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { Token } from '@lumino/coreutils';
+
+import { Signal } from '@lumino/signaling';
+
+import { murmur2 } from 'murmurhash-js';
+
+/**
+ * A class to hash code.
```

Comment:
How about something more specific?

```suggestion
 * A class that holds debugger configuration for a kernel.
```

Reply:
It makes sense